src/attributes.jl

```diff
@@ -337,6 +337,17 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    LogLevel(value::Int)
+
+An optimizer attribute for level of verbosity of the optimizer. `0` means
```

Comment:
Which option has supremacy if a user sets log level here and also a solver-dependent version?

Reply:
Good question, the one set last I'd say.